crates/transaction-emitter-lib/src/emitter/mod.rs

```diff
@@ -188,6 +191,15 @@ pub struct EmitJobRequest {
     tps_wait_after_expiration_secs: Option<u64>,
 
     account_minter_seed: Option<[u8; 32]>,
+
+    // Arguments for Keyless Load Testing
+    keyless_ephem_secret_key: Option<[u8; 32]>,
+
+    proof_file_path: Option<String>,
+
+    epk_expiry_date_secs: Option<u64>,
+
+    keyless_jwt: Option<String>,
```

Comment:
you can remove this if it's part of AccountType

Reply:
couldn't restructure into the enum